pandas/core/reshape/tile.py

```diff
@@ -376,9 +376,8 @@ def _bins_to_cuts(
     if len(unique_bins) < len(bins) and len(bins) != 2:
         if duplicates == "raise":
             raise ValueError(
-                "Bin edges must be unique: {bins!r}.\nYou "
-                "can drop duplicate edges by setting "
-                "the 'duplicates' kwarg".format(bins=bins)
+                f"Bin edges must be unique: {repr(bins)}.\n "
```

Comment:
Should I replace the ```\n``` with a ```;```?

Reply:
is there an extra space here now before the You?

Reply:
@simonjayhawkins Yes, you're absolutely right.